### DISCUSSION TITLE
Update branch shown for a PR in a repository I didn't own

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#133384] [Mobile] [answered]
GitHub Mobile beta for Android 

I got a notification about  https://github.com/radis/radis/pull/667 in the GitHub Mobile application.

When I scrolled down, I saw:
![Screenshot_20240724-080437.png](https://github.com/user-attachments/assets/507f3bac-70cb-414d-9e26-b31f2da913e6)

Note that when I opened the PR in Chrome, it showed that I couldn't update the branch -- amongst other things, it had been merged.

**Comment by iskae** [comment]
Hi @jsoref 👋 

Thanks for the feedback! 

> amongst other things, it had been merged

That sounds like a caching issue on our part. Was the PR open before navigating to the PR through the notification? We are sadly not updating the issues or prs in real time for now, but we are planning for improvements 🙇 

Also, we are planning to update our merge box to be more aligned with the web soon too. Please stay tuned for our future updates 😊

**Reply by jsoref** [reply]
I'm more interested in the fact that it suggested "update branch" where I'm not an owner of the repository.

**Reply by iskae** [reply, marked as answer]
Oh that is a great catch indeed ❤️ 

Let me create a bug report and try to fix it as soon as possible 🙇 

Thanks once again!
